### DISCUSSION TITLE
How would you see the speed relative to performance of a model?

### DISCUSSION THREAD

**Discussion by MrVolts** [embeddings-benchmark/mteb#176] [General]
I noticed in your article: https://huggingface.co/blog/mteb you had a graph showing speed and performance; I was unsure how to access this, but with up-to-date data. Help would be much appreciated as speed with good peformance is crucial for my application.

**Comment by Muennighoff** [comment]
The benchmark was done using this code: https://github.com/embeddings-benchmark/mtebscripts/blob/main/run_benchmark.py
Lmk if you rerun it with new models - would be great to see those results
